### PR TITLE
Add XC 9.3.0

### PR DESCRIPTION
**Pull request by joostmeijles**
Please note that the XC 9.3 tests use by default process isolation as running 15 containers with VM is (at least for my PC with 32GB memory) too resource heavy (see https://docs.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/system-requirements#memory-requirements). Low memory causes problems with starting and stopping containers (see also https://github.com/moby/moby/issues/37429).

VM isolation for XC can be set using `ISOLATION=hyperv` in `.env`.

### CODE REVIEW

**Comment by joostmeijles** [comment]
> can you add build flag into buid.ps1 -IncludeXc
> ask @michaellwest for support @joostmeijles

Isn't that already in place here? https://github.com/Sitecore/docker-images/blob/c46dc561508dadab323de4cd4eb7dbc549736c45/Build.ps1#L24

**Comment by michaellwest** [comment]
@joostmeijles Does it still only include vanilla, SPE, and SXA images? On this line you can see that the SPE and SXA tags are configured. I don't think you would need to change anything until JSS (if ever) get added.

https://github.com/joostmeijles/docker-images/blob/xc_93/Build.ps1#L114

**Comment by joostmeijles** [comment]
Yes, same images included as for XC 9.2

File: windows/9.3.0/sitecore-assets/tools/entrypoints/sitecore-xc-engine/Production.ps1 [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    }
+
+    Start-Sleep -Milliseconds 500
+}
+
+# wait for application pool to stop
+Wait-WebItemState -IISPath "IIS:\AppPools\DefaultAppPool" -State "Stopped"
+
+# Set connection string details
+c:/tools/scripts/Set-CommerceEngineConnectionString.ps1 -userName $env:SQL_USER -password $env:SQL_PASSWORD -server $env:SQL_HOST -globalDatabaseName $env:SQL_COMMERCE_GLOBAL_DB_NAME -sharedEnvironmentDatabaseName $env:SQL_COMMERCE_SHAREDENVIRONMENT_DB_NAME
```
Comment (reyrahadian) [inline windows/9.3.0/sitecore-assets/tools/entrypoints/sitecore-xc-engine/Production.ps1]: please remove this line as it's pointing to a non-existing powershell script. There's an open PR for this to fix the 9.2 XC entrypoint script https://github.com/Sitecore/docker-images/pull/223

Comment (bplasmeijer) [reply windows/9.3.0/sitecore-assets/tools/entrypoints/sitecore-xc-engine/Production.ps1]: @joostmeijles can you verify #233, and add into #230

Comment (joostmeijles) [reply windows/9.3.0/sitecore-assets/tools/entrypoints/sitecore-xc-engine/Production.ps1]: Verified and now included fix from #223

File: windows/tests/9.3.x/README.md [hunk 2]
```diff
@@ -1,3 +1,57 @@
 # Tests
 
-...
\ No newline at end of file
+# Running XC
```
Comment (bplasmeijer) [inline windows/tests/9.3.x/README.md]: add license info, both license keys environments variables are required to run XC

**Pull request merged**